qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -623,3 +625,110 @@ def test_help(self):
         o = self.get_cephfs_shell_cmd_output("help")
 
         log.info("output:\n{}".format(o))
+
+class TestConfReading(TestCephFSShell):
+    def test_reading_conf(self):
+        """
+        Read conf without duplicate sections/options.
+        """
+        from configparser import ConfigParser
+
+        self.ceph_cluster.set_ceph_conf('cephfs-shell', 'debug', 'False')
+        output = self.mount_a.run_shell(['cat', path.abspath(
+            self.mount_a.config_path)])
+
+        cp = ConfigParser(strict=False)
+        cp.read(self.mount_a.config_path)
+        log.info('cp.sections() -')
+        log.info(cp.sections())
+        log.info('cp.has_section(\'cephfs-shell\') = ', str(cp.has_section('cephfs-shell')))
+        if cp.has_section('cephfs-shell'):
+            log.info('cp.items(\'cephfs-shell\') -')
+            log.info(cp.items('cephfs-shell'))
+        optval = str(cp['cephfs-shell']['debug'])
+
+        output = self.get_cephfs_shell_cmd_output('set debug')
```

Comment:
`get debug`? Explain what this command does.

Reply:
This is a (cephfs) shell command that would enable shell to print tracebacks along with the error messages.